config-overrides.js

```diff
@@ -1,39 +1,49 @@
-const CopyWebpackPlugin = require('copy-webpack-plugin')
-const WriteFilePlugin = require('write-file-webpack-plugin')
-
-const isEqual = require("lodash/isEqual")
+const isEqual = require('lodash/isEqual')
+const webpack = require('webpack')
+const DeadCodePlugin = require('webpack-deadcode-plugin')
 
 module.exports = function override(config, env) {
-  if (!config.plugins) {
-    config.plugins = []
-  }
+  config.plugins.push(
+    new webpack.ProvidePlugin({
+      process: 'process/browser',
+    }),
+  )
+
+  config.plugins.push(
+    new DeadCodePlugin({
+      exclude: [
+        'node_modules/**/*',
+        'dist/**/*',
+        'README.md',
+        'codegen.ts',
+        'config-overrides.js',
+        'jest.config.js',
+        'nginx.conf',
+        'package-lock.json',
+        'public/robots.txt',
+      ],
+    }),
+  )
 
   const updatedRules = config.module.rules.filter(
-      (rule) => !isEqual(rule, { parser: { requireEnsure: false } })
+    (rule) => !isEqual(rule, { parser: { requireEnsure: false } }),
   )
-  config.module.rules = updatedRules;
-
-  config.plugins.push(new CopyWebpackPlugin({
-    patterns: [{
-      from: 'node_modules/\@streamr/quick-dijkstra-wasm/dijkstraengine.wasm',
-      to: 'static/js',
-    }],
-  }))
-  config.plugins.push(new WriteFilePlugin())
-
-  config.module.rules.push({
-    test: /dijkstraengine\.js$/,
-    loader: "exports-loader",
-    options: {
-      exports: "Module",
-    }
-  })
-
-  config.output.futureEmitAssets = false
+
+  config.module.rules = updatedRules
 
   if (process.env.REACT_APP_PUBLIC_URL) {
     config.output.publicPath = `${process.env.REACT_APP_PUBLIC_URL}/`
   }
 
+  // Drop `ModuleScopePlugin`.
+  config.resolve.plugins = []
+
+  config.resolve.fallback = {
```

Comment:
These fallbacks and the `process/browser` are required by the new `@streamr/utils`.